### ISSUE TITLE
After upgrading to Marlin 2.0.9.3 - Values not pulled from EEPROM

### ISSUE THREAD

**Issue by brandonclyon** [SergiuToporjinschi/OctoPrint-CalibrationTools#27]
I flashed from Marlin 2.0.7.2 to 2.0.9.2 yesterday.

Since then the plugin will not automatically pull from EEPROM, checking terminal the commands are issues and results given but the are not imported into the plugin interface.

**Comment by SergiuToporjinschi** [comment]
Hi @brandonclyon, What do you mean by that, when you push refresh buttons?

**Comment by brandonclyon** [comment]
Correct, in the webUI I click to pull the current values for either PID or eStep, the terminal shows that the command is issued and gives a response but those values are not transferred to the plugin fields.

EDIT: I've upload my conf.h and conf_adv.h to my github

**Comment by Loukey42** [comment]
Similarly, I had it working perfectly with Creality motherboard 4.2.2 running stock firmware then Creality 4.2.7 running both stock and custom (2.0.9.3 and 2.0.bugfix) but just changed over to a SRK E3 mini v2 running 2.0.9.3 and it will not pull any values.  I've tried a few times and can't get the plugin to pull any values.  It runs the update correctly.  I can supply any logs which could be helpful.

**Comment by skl111** [comment]
Firmware Name: Marlin bugfix-2.0.x (Feb 6 2022 20:52:55)
Source Code Url: github.com/MarlinFirmware/Marlin
Protocol Version: 1.0
Machine Type: Ender 5 Pro BTT SKR Mini E3 V2.0
the HE PID Tune works but the Bed does not. Pressing the button does not even bring up the pop up or give and error message

**Comment by jthomeo** [comment]
Any updates on this issue? How the plugin get the data to populate it's fields?

**Comment by IsmaelPR1977** [comment]
Same issue, Rambo Mini 1.3a Prusa MK2.5, values will not load nor do I have the option to refresh/pull values from board.

**Comment by netmilk** [comment]
Hotfixed in my PR #36. [Here](https://github.com/netmilk/OctoPrint-CalibrationTools/archive/refs/heads/main.zip) you can download the .`zip` of my fork and install it manually until it's merged.
